Source/Core/DolphinQt2/CMakeLists.txt

```diff
@@ -28,6 +28,7 @@ set(SRCS
   GameList/GameTracker.cpp
   GameList/ListProxyModel.cpp
   GameList/TableDelegate.cpp
+  Settings/GeneralPane.cpp
```

Comment:
But, the list is already alphabetically ordered?
Notice it goes:
```
Config/SettingsWindow.cpp
GameList/GameFile.cpp
```
So between folders, ordering isn't preserved.
Therefore
```
GameList/TableDelegate.cpp
Settings/GeneralPane.cpp
```

is well ordered too, and will probably be neater when the other panes are merged.